db/data/autotest_files/jupyter/student_files/submission.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6920ccbe",
+   "metadata": {},
+   "source": [
+    "Welcome to your assignment! Do the questions and write the answers in the code cells provided below"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7614a548",
+   "metadata": {},
+   "source": [
+    "Here's a bit of setup for you"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "db8f5c50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pylab inline\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75457f5e",
+   "metadata": {},
+   "source": [
+    "Question 1: Create a 100 x 100 nump array filled with random floats and assign it to the variable `a`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "97df1cad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = np.random.uniform(size=(100,99))"
```

Comment:
It would be great to add a comment above this line saying that the shape is deliberately incorrect to illustrate failing tests.